index.html

```diff
@@ -10,23 +10,215 @@
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+     <div class="project" id="project3">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Tonic</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project4">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>  
+  </section>
+
+  <section class="about-me">
+    <div class="primary-text-container">
+      <h1 class="primary-text">About<br>Myself</h1>
+    </div>
+    <div class="secondary-text">
+      <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
     <div class="connect-text">LET'S CONNECT</div>
     <div class="socialmedia-icons">
-      <a class="twitter-icon"></a>
-      <a class="linkedin-icon"></a>
-      <a class="m-icon"></a>
-      <a class="cat-icon"></a>
-      <a class="fingers-icon"></a>
+      <ul>
+        <li class="twitter-icon"><a href=""></a></li>
+        <li class="linkedin-icon"><a href=""></a></li>
+        <li class="m-icon"><a href=""></a></li>
+        <li class="cat-icon"><a href=""></a></li>
+        <li class="fingers-icon"><a href=""></a></li>
+      </ul>
     </div>
-  </div>
+    <div class="resume-access">Get my Resume</div>
```

Comment:
- [ ] I think it would be better to use `<button>` or `<a>` tag here instead of `div` tag. because this is a link 👌. And please don't forget to follow the button styling: Active, Hover, and Pressed.

![2](https://user-images.githubusercontent.com/45231276/176837061-facdfea9-a844-49bc-bed7-93be8ab7b49c.PNG)